### PR TITLE
💇‍♂️ Bootstrap styling with styled-components & Reach UI

### LINKED ISSUE #10: Bootstrap styling
Add global styles and themes, along with a few example components that use Reach UI. We are using Reach UI to ensure accessibility in this project.

### PR DESCRIPTION
**Pull request by Qrtn**
## Status:
:rocket: Ready

## Description

Bootstraps much of the styling infra by:

- Adding `styled-components`
- Adding global styles and a theme object containing palette colors
- Example UI components such as
  - PageTabs, which is based off of Reach UI's Tabs
  - Button, based off of HTML button
- Reach UI will be used for applicable UI components to ensure accessibility 

Fixes #10

## Screenshots

image

### CODE REVIEW

**Review by SirajChokshi — APPROVED** [review]
nice work 😎

File: src/react-app-env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (laurenho025) [inline src/react-app-env.d.ts:1]: What's this file for? Should it be in gitignore?

Comment (Qrtn) [reply src/react-app-env.d.ts:1]: This is a TypeScript declaration file, kind of like a header file in C/C++. It should be committed and is also present in the API wrapper PR.

**Review by laurenho025 — APPROVED** [review]
lgtm

**Pull request merged**